src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -108,7 +108,7 @@ function ReportActionItemSingle(props) {
 
     // If this is a report preview, display names and avatars of both people involved
     let secondaryAvatar = {};
-    const primaryDisplayName = ReportUtils.getDisplayNameForParticipant(actorAccountID);
+    const primaryDisplayName = isWorkspaceActor ? displayName : ReportUtils.getDisplayNameForParticipant(actorAccountID);
```

Comment:
I think this should be `const primaryDisplayName = displayName;`

Reply:
https://github.com/Expensify/App/pull/30342#issuecomment-1792777038

Reply:
Ah, yeah, so just a full revert of that line. Linking your conversation [here](https://github.com/Expensify/App/pull/30342#issuecomment-1792781543) for convenience